src/tree/updater_quantile_hist.h

```diff
@@ -218,13 +334,20 @@ class QuantileHistMaker: public TreeUpdater {
     const DMatrix* p_last_fmat_;
 
     using ExpandQueue =
-        std::priority_queue<ExpandEntry, std::vector<ExpandEntry>,
-                            std::function<bool(ExpandEntry, ExpandEntry)>>;
-    std::unique_ptr<ExpandQueue> qexpand_;
+       std::priority_queue<ExpandEntry, std::vector<ExpandEntry>,
+                           std::function<bool(ExpandEntry, ExpandEntry)>>;
+
+    std::unique_ptr<ExpandQueue> qexpand_loss_guided_;
+    std::vector<ExpandEntry> qexpand_depth_wise_;
+    // key is the node id which should be calculated by SubstractTrick, value is the node which
+    // provides the evidence for substracts
+    std::unordered_map<int, int> nodes_for_substrack_trick;
```

Comment:
Let's use correct spelling:
```suggestion
    std::unordered_map<int, int> nodes_for_subtraction_trick;
```